File: client_manage_interface/src/main/java/cn/edu/aynu/rjxy/management/service/TicketService.java
package cn.edu.aynu.rjxy.management.service;


import cn.edu.aynu.rjxy.pojo.TbTicket;
import com.github.pagehelper.PageInfo;

import java.util.List;

/**
 * 服务层接口
 * @author Administrator
 *
 */
public interface TicketService {

	/**
	 * 返回全部列表
	 * @return
	 */
	public List<TbTicket> findAll();
	
	
	/**
	 * 返回分页列表
	 * @return
	 */
	public PageInfo findPage(int pageNum, int pageSize);


	/**
	 * 增加
	*/
	public void add(TbTicket ticket);


	/**
	 * 修改
	 */
	public void update(TbTicket ticket);


	/**
	 * 根据ID获取实体
	 * @param id
	 * @return
	 */
	public TbTicket findOne(Long ticketId);


	/**
	 * 批量删除
	 * @param ids
	 */
	public void delete(Long[] ticketIds);

	/**
	 * 分页
	 * @param pageNum 当前页 码
	 * @param pageSize 每页记录数
	 * @return
	 */
	public PageInfo findPage(TbTicket ticket, int pageNum, int pageSize);
	
}

File: client_manage_web/src/main/webapp/js/service/ticketService.js
//服务层
app.service('ticketService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../ticket/findAll.do');		
	}
	//分页 
	this.findPage=function(page,rows){
		return $http.get('../ticket/findPage.do?pageNum='+page+'&pageSize='+rows);
	}
	//查询实体
	this.findOne=function(id){
		return $http.get('../ticket/findOne.do?id='+id);
	}
	//增加 
	this.add=function(entity){
		return  $http.post('../ticket/add.do',entity );
	}
	//修改 
	this.update=function(entity){
		return  $http.post('../ticket/update.do',entity );
	}
	//删除
	this.dele=function(ids){
		return $http.get('../ticket/delete.do?ids='+ids);
	}
	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../ticket/search.do?pageNum='+page+"&pageSize="+rows, searchEntity);
	}    	
});